Can't install with pip

This is the error report(cmake is already installed): 
```C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\bin>python -m pip install opentimelineio
C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\bin>python -m pip install opentimelineio
Collecting opentimelineio
  Using cached OpenTimelineIO-0.12.0.tar.gz (2.0 MB)
Requirement already satisfied: pyaaf2==1.2.0 in c:\users\user\documents\build_windows_x64_vc16_release\bin\release\2.83\python\lib\site-packages (from opentimelineio) (1.2.0)
Installing collected packages: opentimelineio
    Running setup.py install for opentimelineio ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\bin\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-w6q_74kv\\opentimelineio\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-w6q_74kv\\opentimelineio\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\user\AppData\Local\Temp\pip-record-qe4za2pc\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\Include\opentimelineio'
         cwd: C:\Users\user\AppData\Local\Temp\pip-install-w6q_74kv\opentimelineio\
    Complete output (184 lines):
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.7
    creating build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\exceptions.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\hooks.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\media_linker.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\opentime.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\test_utils.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\__init__.py -> build\lib.win-amd64-3.7\opentimelineio
    creating build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\adapter.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\cmx_3600.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\fcp_xml.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\otio_json.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    creating build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\filter.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\stack_algo.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\timeline_algo.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\track_algo.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    creating build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\autogen_plugin_documentation.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\autogen_serialized_datamodel.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\console_utils.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otiocat.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otioconvert.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otiopluginfo.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otiostat.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\console
    creating build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\composable.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\composition.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\item.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\mediaReference.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\_core_utils.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\core
    creating build\lib.win-amd64-3.7\opentimelineio\plugins
    copying src\py-opentimelineio\opentimelineio\plugins\manifest.py -> build\lib.win-amd64-3.7\opentimelineio\plugins
    copying src\py-opentimelineio\opentimelineio\plugins\python_plugin.py -> build\lib.win-amd64-3.7\opentimelineio\plugins
    copying src\py-opentimelineio\opentimelineio\plugins\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\plugins
    creating build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\clip.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\effect.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\external_reference.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\foo.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\generator_reference.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\marker.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\schemadef.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\serializable_collection.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\stack.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\timeline.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\track.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\transition.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    creating build\lib.win-amd64-3.7\opentimelineio\schemadef
    copying src\py-opentimelineio\opentimelineio\schemadef\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\schemadef
    creating build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\console.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\details_widget.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\ruler_widget.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\settings.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\timeline_widget.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\track_widgets.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\__init__.py -> build\lib.win-amd64-3.7\opentimelineview
    creating build\lib.win-amd64-3.7\opentimelineio_contrib
    copying contrib\opentimelineio_contrib\__init__.py -> build\lib.win-amd64-3.7\opentimelineio_contrib
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\advanced_authoring_format.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\ale.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\burnins.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\extern_maya_sequencer.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\extern_rv.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\fcpx_xml.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\ffmpeg_burnins.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\hls_playlist.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\kdenlive.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\maya_sequencer.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\rv.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\xges.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\__init__.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\adapters\aaf_adapter
    copying contrib\opentimelineio_contrib\adapters\aaf_adapter\aaf_writer.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters\aaf_adapter
    copying contrib\opentimelineio_contrib\adapters\aaf_adapter\__init__.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters\aaf_adapter
    running egg_info
    writing OpenTimelineIO.egg-info\PKG-INFO
    writing dependency_links to OpenTimelineIO.egg-info\dependency_links.txt
    writing entry points to OpenTimelineIO.egg-info\entry_points.txt
    writing requirements to OpenTimelineIO.egg-info\requires.txt
    writing top-level names to OpenTimelineIO.egg-info\top_level.txt
    reading manifest file 'OpenTimelineIO.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    warning: no files found matching 'CHANGELOG.md'
    no previously-included directories found matching 'docs'
    warning: no previously-included files found matching '.gitmodules'
    warning: no previously-included files matching '*.git' found under directory 'src'
    warning: no previously-included files found matching '.readthedocs.yml'
    warning: no previously-included files found matching 'readthedocs-conda.yml'
    warning: no previously-included files found matching '.codecov.yml'
    warning: no previously-included files found matching '.gitlab-ci.yml'
    warning: no previously-included files found matching '.travis.yml'
    warning: no previously-included files found matching '*.pdf'
    warning: no previously-included files found matching 'CODE_OF_CONDUCT.md'
    warning: no previously-included files found matching 'CONTRIBUTING.md'
    warning: no previously-included files found matching 'contrib\opentimelineio_contrib\adapters\Makefile'
    warning: no previously-included files found matching 'Makefile'
    no previously-included directories found matching 'contrib\opentimelineio_contrib\adapters\tests'
    no previously-included directories found matching 'tests'
    no previously-included directories found matching 'src\deps\pybind11\tools\clang'
    no previously-included directories found matching 'src\deps\rapidjson\thirdparty'
    writing manifest file 'OpenTimelineIO.egg-info\SOURCES.txt'
    copying src\py-opentimelineio\opentimelineio\adapters\builtin_adapters.plugin_manifest.json -> build\lib.win-amd64-3.7\opentimelineio\adapters
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\PACKAGE -> build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\example_otio_reader_plugin.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\otio_reader.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\adapters\contrib_adapters.plugin_manifest.json -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    running build_ext
    -- Building for: Visual Studio 16 2019
    -- The C compiler identification is MSVC 19.23.28106.4
    -- The CXX compiler identification is MSVC 19.23.28106.4
    -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe
    -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe -- works
    -- Detecting C compiler ABI info
    -- Detecting C compiler ABI info - done
    -- Detecting C compile features
    -- Detecting C compile features - done
    -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe
    -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe -- works
    -- Detecting CXX compiler ABI info
    -- Detecting CXX compiler ABI info - done
    -- Detecting CXX compile features
    -- Detecting CXX compile features - done
    fatal: not a git repository (or any of the parent directories): .git
    -- Found PythonInterp: C:/Users/user/Documents/build_windows_x64_vc16_Release/bin/Release/2.83/python/bin/python.exe (found version "3.7.4")
    CMake Error at src/deps/pybind11/tools/FindPythonLibsNew.cmake:158 (message):
      Python libraries not found
    Call Stack (most recent call first):
      src/deps/pybind11/tools/pybind11Tools.cmake:16 (find_package)
      src/deps/pybind11/CMakeLists.txt:33 (include)


    -- Configuring incomplete, errors occurred!
    See also "C:/Users/user/AppData/Local/Temp/pip-install-w6q_74kv/opentimelineio/build/temp.win-amd64-3.7/Release/CMakeFiles/CMakeOutput.log".
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-install-w6q_74kv\opentimelineio\setup.py", line 466, in <module>
        **PROJECT_METADATA
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\site-packages\setuptools\__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\core.py", line 148, in setup
        dist.run_commands()
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 985, in run_command
        cmd_obj.run()
      File "C:\Users\user\AppData\Local\Temp\pip-install-w6q_74kv\opentimelineio\setup.py", line 140, in run
        install.run(self)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\site-packages\setuptools\command\install.py", line 61, in run
        return orig.install.run(self)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\command\install.py", line 545, in run
        self.run_command('build')
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 985, in run_command
        cmd_obj.run()
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\command\build.py", line 135, in run
        self.run_command(cmd_name)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 985, in run_command
        cmd_obj.run()
      File "C:\Users\user\AppData\Local\Temp\pip-install-w6q_74kv\opentimelineio\setup.py", line 185, in run
        self.build()
      File "C:\Users\user\AppData\Local\Temp\pip-install-w6q_74kv\opentimelineio\setup.py", line 216, in build
        env=env
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\subprocess.py", line 347, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['cmake', 'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-w6q_74kv\\opentimelineio', '-DPYTHON_EXECUTABLE=C:\\Users\\user\\Documents\\build_windows_x64_vc16_Release\\bin\\Release\\2.83\\python\\bin\\python.exe', '-DOTIO_PYTHON_INSTALL:BOOL=ON', '-DOTIO_PYTHON_OTIO_DIR=C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-w6q_74kv\\opentimelineio\\build\\lib.win-amd64-3.7\\opentimelineio\\', '-DCMAKE_INSTALL_PREFIX=C:\\Users\\user\\Documents\\build_windows_x64_vc16_Release\\bin\\Release\\2.83\\python\\Lib\\site-packages\\opentimelineio\\cxx-libs', '-DOTIO_CXX_NOINSTALL:BOOL=ON', '-A', 'x64']' returned non-zero exit status 1.
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\bin\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-w6q_74kv\\opentimelineio\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-w6q_74kv\\opentimelineio\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\user\AppData\Local\Temp\pip-record-qe4za2pc\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\Include\opentimelineio' Check the logs for full command output.
```

Python 2.7 + VS2019 + cmake 3.13 fails as well. Omitting lots of errors, here's the first problem.

```
   -- Found PythonLibs: C:/Python27/libs/Python27.lib
    -- pybind11 v2.3.dev0
    CMake Error at src/opentime/CMakeLists.txt:17 (install):
      install Library TARGETS given no DESTINATION!


    CMake Error at src/opentimelineio/CMakeLists.txt:80 (install):
      install Library TARGETS given no DESTINATION!
```

I'm sorry. I'm still getting errors:
```C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\bin>python -m pip install opentimelineio
Collecting opentimelineio
  Using cached OpenTimelineIO-0.12.0.tar.gz (2.0 MB)
Requirement already satisfied: pyaaf2==1.2.0 in c:\users\user\documents\build_windows_x64_vc16_release\bin\release\2.83\python\lib\site-packages (from opentimelineio) (1.2.0)
Installing collected packages: opentimelineio
    Running setup.py install for opentimelineio ... error
    ERROR: Command errored out with exit status 1:
     command: 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\bin\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d9p791wd\\opentimelineio\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d9p791wd\\opentimelineio\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\user\AppData\Local\Temp\pip-record-dufrbuw1\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\Include\opentimelineio'
         cwd: C:\Users\user\AppData\Local\Temp\pip-install-d9p791wd\opentimelineio\
    Complete output (184 lines):
    running install
    running build
    running build_py
    creating build
    creating build\lib.win-amd64-3.7
    creating build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\exceptions.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\hooks.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\media_linker.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\opentime.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\test_utils.py -> build\lib.win-amd64-3.7\opentimelineio
    copying src\py-opentimelineio\opentimelineio\__init__.py -> build\lib.win-amd64-3.7\opentimelineio
    creating build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\adapter.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\cmx_3600.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\fcp_xml.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\otio_json.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    copying src\py-opentimelineio\opentimelineio\adapters\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\adapters
    creating build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\filter.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\stack_algo.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\timeline_algo.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\track_algo.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    copying src\py-opentimelineio\opentimelineio\algorithms\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\algorithms
    creating build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\autogen_plugin_documentation.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\autogen_serialized_datamodel.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\console_utils.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otiocat.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otioconvert.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otiopluginfo.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\otiostat.py -> build\lib.win-amd64-3.7\opentimelineio\console
    copying src\py-opentimelineio\opentimelineio\console\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\console
    creating build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\composable.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\composition.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\item.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\mediaReference.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\_core_utils.py -> build\lib.win-amd64-3.7\opentimelineio\core
    copying src\py-opentimelineio\opentimelineio\core\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\core
    creating build\lib.win-amd64-3.7\opentimelineio\plugins
    copying src\py-opentimelineio\opentimelineio\plugins\manifest.py -> build\lib.win-amd64-3.7\opentimelineio\plugins
    copying src\py-opentimelineio\opentimelineio\plugins\python_plugin.py -> build\lib.win-amd64-3.7\opentimelineio\plugins
    copying src\py-opentimelineio\opentimelineio\plugins\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\plugins
    creating build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\clip.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\effect.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\external_reference.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\foo.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\generator_reference.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\marker.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\schemadef.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\serializable_collection.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\stack.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\timeline.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\track.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\transition.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    copying src\py-opentimelineio\opentimelineio\schema\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\schema
    creating build\lib.win-amd64-3.7\opentimelineio\schemadef
    copying src\py-opentimelineio\opentimelineio\schemadef\__init__.py -> build\lib.win-amd64-3.7\opentimelineio\schemadef
    creating build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\console.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\details_widget.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\ruler_widget.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\settings.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\timeline_widget.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\track_widgets.py -> build\lib.win-amd64-3.7\opentimelineview
    copying src\opentimelineview\__init__.py -> build\lib.win-amd64-3.7\opentimelineview
    creating build\lib.win-amd64-3.7\opentimelineio_contrib
    copying contrib\opentimelineio_contrib\__init__.py -> build\lib.win-amd64-3.7\opentimelineio_contrib
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\advanced_authoring_format.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\ale.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\burnins.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\extern_maya_sequencer.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\extern_rv.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\fcpx_xml.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\ffmpeg_burnins.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\hls_playlist.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\kdenlive.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\maya_sequencer.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\rv.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\xges.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    copying contrib\opentimelineio_contrib\adapters\__init__.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\adapters\aaf_adapter
    copying contrib\opentimelineio_contrib\adapters\aaf_adapter\aaf_writer.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters\aaf_adapter
    copying contrib\opentimelineio_contrib\adapters\aaf_adapter\__init__.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters\aaf_adapter
    running egg_info
    writing OpenTimelineIO.egg-info\PKG-INFO
    writing dependency_links to OpenTimelineIO.egg-info\dependency_links.txt
    writing entry points to OpenTimelineIO.egg-info\entry_points.txt
    writing requirements to OpenTimelineIO.egg-info\requires.txt
    writing top-level names to OpenTimelineIO.egg-info\top_level.txt
    reading manifest file 'OpenTimelineIO.egg-info\SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    warning: no files found matching 'CHANGELOG.md'
    no previously-included directories found matching 'docs'
    warning: no previously-included files found matching '.gitmodules'
    warning: no previously-included files matching '*.git' found under directory 'src'
    warning: no previously-included files found matching '.readthedocs.yml'
    warning: no previously-included files found matching 'readthedocs-conda.yml'
    warning: no previously-included files found matching '.codecov.yml'
    warning: no previously-included files found matching '.gitlab-ci.yml'
    warning: no previously-included files found matching '.travis.yml'
    warning: no previously-included files found matching '*.pdf'
    warning: no previously-included files found matching 'CODE_OF_CONDUCT.md'
    warning: no previously-included files found matching 'CONTRIBUTING.md'
    warning: no previously-included files found matching 'contrib\opentimelineio_contrib\adapters\Makefile'
    warning: no previously-included files found matching 'Makefile'
    no previously-included directories found matching 'contrib\opentimelineio_contrib\adapters\tests'
    no previously-included directories found matching 'tests'
    no previously-included directories found matching 'src\deps\pybind11\tools\clang'
    no previously-included directories found matching 'src\deps\rapidjson\thirdparty'
    writing manifest file 'OpenTimelineIO.egg-info\SOURCES.txt'
    copying src\py-opentimelineio\opentimelineio\adapters\builtin_adapters.plugin_manifest.json -> build\lib.win-amd64-3.7\opentimelineio\adapters
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv
    creating build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\PACKAGE -> build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\example_otio_reader_plugin.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\application_plugins\rv\example_otio_reader\otio_reader.py -> build\lib.win-amd64-3.7\opentimelineio_contrib\application_plugins\rv\example_otio_reader
    copying contrib\opentimelineio_contrib\adapters\contrib_adapters.plugin_manifest.json -> build\lib.win-amd64-3.7\opentimelineio_contrib\adapters
    running build_ext
    -- Building for: Visual Studio 16 2019
    -- The C compiler identification is MSVC 19.23.28106.4
    -- The CXX compiler identification is MSVC 19.23.28106.4
    -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe
    -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe -- works
    -- Detecting C compiler ABI info
    -- Detecting C compiler ABI info - done
    -- Detecting C compile features
    -- Detecting C compile features - done
    -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe
    -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.23.28105/bin/Hostx64/x64/cl.exe -- works
    -- Detecting CXX compiler ABI info
    -- Detecting CXX compiler ABI info - done
    -- Detecting CXX compile features
    -- Detecting CXX compile features - done
    fatal: not a git repository (or any of the parent directories): .git
    -- Found PythonInterp: C:/Users/user/Documents/build_windows_x64_vc16_Release/bin/Release/2.83/python/bin/python.exe (found version "3.7.4")
    CMake Error at src/deps/pybind11/tools/FindPythonLibsNew.cmake:158 (message):
      Python libraries not found
    Call Stack (most recent call first):
      src/deps/pybind11/tools/pybind11Tools.cmake:16 (find_package)
      src/deps/pybind11/CMakeLists.txt:33 (include)


    -- Configuring incomplete, errors occurred!
    See also "C:/Users/user/AppData/Local/Temp/pip-install-d9p791wd/opentimelineio/build/temp.win-amd64-3.7/Release/CMakeFiles/CMakeOutput.log".
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-install-d9p791wd\opentimelineio\setup.py", line 466, in <module>
        **PROJECT_METADATA
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\site-packages\setuptools\__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\core.py", line 148, in setup
        dist.run_commands()
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 985, in run_command
        cmd_obj.run()
      File "C:\Users\user\AppData\Local\Temp\pip-install-d9p791wd\opentimelineio\setup.py", line 140, in run
        install.run(self)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\site-packages\setuptools\command\install.py", line 61, in run
        return orig.install.run(self)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\command\install.py", line 545, in run
        self.run_command('build')
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 985, in run_command
        cmd_obj.run()
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\command\build.py", line 135, in run
        self.run_command(cmd_name)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\cmd.py", line 313, in run_command
        self.distribution.run_command(command)
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\distutils\dist.py", line 985, in run_command
        cmd_obj.run()
      File "C:\Users\user\AppData\Local\Temp\pip-install-d9p791wd\opentimelineio\setup.py", line 185, in run
        self.build()
      File "C:\Users\user\AppData\Local\Temp\pip-install-d9p791wd\opentimelineio\setup.py", line 216, in build
        env=env
      File "C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\lib\subprocess.py", line 347, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['cmake', 'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d9p791wd\\opentimelineio', '-DPYTHON_EXECUTABLE=C:\\Users\\user\\Documents\\build_windows_x64_vc16_Release\\bin\\Release\\2.83\\python\\bin\\python.exe', '-DOTIO_PYTHON_INSTALL:BOOL=ON', '-DOTIO_PYTHON_OTIO_DIR=C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d9p791wd\\opentimelineio\\build\\lib.win-amd64-3.7\\opentimelineio\\', '-DCMAKE_INSTALL_PREFIX=C:\\Users\\user\\Documents\\build_windows_x64_vc16_Release\\bin\\Release\\2.83\\python\\Lib\\site-packages\\opentimelineio\\cxx-libs', '-DOTIO_CXX_NOINSTALL:BOOL=ON', '-A', 'x64']' returned non-zero exit status 1.
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\bin\python.exe' -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d9p791wd\\opentimelineio\\setup.py'"'"'; __file__='"'"'C:\\Users\\user\\AppData\\Local\\Temp\\pip-install-d9p791wd\\opentimelineio\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\user\AppData\Local\Temp\pip-record-dufrbuw1\install-record.txt' --single-version-externally-managed --compile --install-headers 'C:\Users\user\Documents\build_windows_x64_vc16_Release\bin\Release\2.83\python\Include\opentimelineio' Check the logs for full command output.```

--- a/.github/workflows/python-package.yml
+++ b/.github/workflows/python-package.yml
@@ -66,30 +66,66 @@ jobs:
         fail_ci_if_error: true
 
   package_wheels:
-    name: Package wheels on ${{ matrix.os }}
     needs: build
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, macOS-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
         python-build: [cp27*, cp37*, cp38*]
+        exclude:
+          # Python 2.7 on Windows requires a workaround for C++11 support,
+          # built separately below
+          - os: windows-latest
+            python-build: cp27*
 
     steps:
       - uses: actions/checkout@v2
 
-      # Used to host cibuildwheel
-      - uses: actions/setup-python@v2
-
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==1.10.0
-
       - name: Build wheels
-        run: python -m cibuildwheel --output-dir wheelhouse
-        # TODO: Solve for the 32-bit errors in windows python 2.7:
-        #   https://github.com/pybind/cmake_example/blob/master/.github/workflows/wheels.yml#L66
+        uses: pypa/cibuildwheel@v1.12.0
+        with:
+          output-dir: wheelhouse
         env:
-          CIBW_SKIP: cp27-win*
           CIBW_BUILD: ${{ matrix.python-build }}
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: wheels
+          path: ./wheelhouse/*.whl
+
+  # Windows 2.7 (requires workaround for MSVC 2008 replacement)
+  package_wheels_win:
+    needs: build
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        os: [windows-latest]
+        python-build: [cp27*]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ilammy/msvc-dev-cmd@v1
+
+      - name: Build 64-bit wheel
+        uses: pypa/cibuildwheel@v1.12.0
+        with:
+          output-dir: wheelhouse
+        env:
+          CIBW_BUILD: cp27-win_amd64
+          DISTUTILS_USE_SDK: 1
+          MSSdk: 1
+
+      - uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: x86
+
+      - name: Build 32-bit wheel
+        uses: pypa/cibuildwheel@v1.12.0
+        env:
+          CIBW_BUILD: cp27-win32
+          DISTUTILS_USE_SDK: 1
+          MSSdk: 1
 
       - uses: actions/upload-artifact@v2
         with:

--- a/.github/workflows/python-package.yml
+++ b/.github/workflows/python-package.yml
@@ -72,14 +72,24 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-build: [cp27*, cp37*, cp38*]
-        exclude:
-          # Python 2.7 on Windows requires a workaround for C++11 support,
-          # built separately below
-          - os: windows-latest
-            python-build: cp27*
-
     steps:
       - uses: actions/checkout@v2
+
+      # cibuildwheel 1.12.0 gates Python 2.7 wheels builds
+      # by using two environment variables, DISTUTILS_USE_SDK and MSSdk.
+      # https://cibuildwheel.readthedocs.io/en/1.x/cpp_standards/#windows-and-python-27
+      # Note that normally these are used by setuptools/distutils, but in our case
+      # they are really just used for cibuildwheel as we don't use any of the
+      # setuptools/distutils build tools. Our builds are entirely handled
+      # by CMake. CMake is able to find the right toolchain, thanks to
+      # the -A argument that we specify in the setup.py to set the
+      # target platform (x86, x64, etc).
+      - name: Set Windows Python 2.7 environment variables
+        if: matrix.python-build == 'cp27*' && runner.os == 'Windows'
+        shell: bash
+        run: |
+          echo "DISTUTILS_USE_SDK=1" >> $GITHUB_ENV
+          echo "MSSdk=1" >> $GITHUB_ENV
 
       - name: Build wheels
         uses: pypa/cibuildwheel@v1.12.0
@@ -87,45 +97,6 @@ jobs:
           output-dir: wheelhouse
         env:
           CIBW_BUILD: ${{ matrix.python-build }}
-
-      - uses: actions/upload-artifact@v2
-        with:
-          name: wheels
-          path: ./wheelhouse/*.whl
-
-  # Windows 2.7 (requires workaround for MSVC 2008 replacement)
-  package_wheels_win:
-    needs: build
-    runs-on: windows-latest
-    strategy:
-      matrix:
-        os: [windows-latest]
-        python-build: [cp27*]
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: ilammy/msvc-dev-cmd@v1
-
-      - name: Build 64-bit wheel
-        uses: pypa/cibuildwheel@v1.12.0
-        with:
-          output-dir: wheelhouse
-        env:
-          CIBW_BUILD: cp27-win_amd64
-          DISTUTILS_USE_SDK: 1
-          MSSdk: 1
-
-      - uses: ilammy/msvc-dev-cmd@v1
-        with:
-          arch: x86
-
-      - name: Build 32-bit wheel
-        uses: pypa/cibuildwheel@v1.12.0
-        env:
-          CIBW_BUILD: cp27-win32
-          DISTUTILS_USE_SDK: 1
-          MSSdk: 1
 
       - uses: actions/upload-artifact@v2
         with:

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -98,10 +98,22 @@ if(OTIO_SHARED_LIBS)
 else()
     message(STATUS "Building static libs")
     set(OTIO_SHARED_OR_STATIC_LIB "STATIC")
-    if (OTIO_PYTHON_INSTALL AND NOT MSVC)
-        # If we're compiling for pybind, we can hide all our symbols, they'll only be called from pybind
-        set(CMAKE_CXX_VISIBILITY_PRESET hidden) 
-        set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
+    if (OTIO_PYTHON_INSTALL)
+        if(MSVC AND Python_VERSION_MAJOR VERSION_LESS 3)
+            # Statically link run-time library (vcruntime and msvcp)
+            # See https://docs.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-160
+            # This allows us to compile OTIO bindings with a newer MSVC version
+            # than the one used by the interpreter where OTIO will be installed.
+            # This is only required for Python < 3 because only these are
+            # compiled with an older compiler (9.0). CPython 3.5+ uses at least
+            # Visual C++ 14.X.
+            # See https://wiki.python.org/moin/WindowsCompilers#Which_Microsoft_Visual_C.2B-.2B-_compiler_to_use_with_a_specific_Python_version_.3F
+            add_compile_options(/MT)
+        else()
+            # If we're compiling for pybind, we can hide all our symbols, they'll only be called from pybind
+            set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+            set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
+        endif()
     endif()
 endif()
 

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -99,6 +99,11 @@ else()
     message(STATUS "Building static libs")
     set(OTIO_SHARED_OR_STATIC_LIB "STATIC")
     if (OTIO_PYTHON_INSTALL)
+        # If we're compiling for pybind, we can hide all our symbols, they'll only be called from pybind
+        # Note that this has no effect on Windows.
+        set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+        set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
+
         if(MSVC AND Python_VERSION_MAJOR VERSION_LESS 3)
             # Statically link run-time library (vcruntime and msvcp)
             # See https://docs.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-160
@@ -109,10 +114,6 @@ else()
             # Visual C++ 14.X.
             # See https://wiki.python.org/moin/WindowsCompilers#Which_Microsoft_Visual_C.2B-.2B-_compiler_to_use_with_a_specific_Python_version_.3F
             add_compile_options(/MT)
-        else()
-            # If we're compiling for pybind, we can hide all our symbols, they'll only be called from pybind
-            set(CMAKE_CXX_VISIBILITY_PRESET hidden)
-            set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
         endif()
     endif()
 endif()

--- a/setup.py
+++ b/setup.py
@@ -27,6 +27,10 @@ import setuptools.command.build_py
 
 SOURCE_DIR = os.path.abspath(os.path.dirname(__file__))
 
+PLAT_TO_CMAKE = {
+    "win32": "Win32",
+    "win-amd64": "x64",
+}
 
 INSTALL_REQUIRES = [
     'pyaaf2~=1.4.0',
@@ -103,8 +107,7 @@ class OTIO_build_ext(setuptools.command.build_ext.build_ext):
         ]
 
         if platform.system() == "Windows":
-            if sys.maxsize > 2**32:
-                cmake_args += ['-A', 'x64']
+            cmake_args += ["-A", PLAT_TO_CMAKE[self.plat_name]]
 
         cxx_coverage = bool(os.environ.get("OTIO_CXX_COVERAGE_BUILD"))
         if cxx_coverage and not os.environ.get("OTIO_CXX_BUILD_TMP_DIR"):

